### PR TITLE
Feature/spa to angular10

### PR DESCRIPTION
**Pull request by cdemiguel**
Update SPA to Angular version 10:
 - Updated all required packages to update ng version
 - Fix bug on catalog filtering
 - Correct endpoint into appsettings
 - Updated to Angular version 10

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-architecture/eShopOnContainers?pullRequest=1549) <br/>All CLA requirements met.

File: src/Web/WebSPA/Client/modules/catalog/catalog.component.ts [hunk 1]
```diff
@@ -56,6 +56,9 @@ export class CatalogComponent implements OnInit {
 
     onFilterApplied(event: any) {
         event.preventDefault();
+        
+        this.brandSelected = this.brandSelected && this.brandSelected.toString() != "null" ? this.brandSelected : null;
```
Comment (mvelosop) [inline src/Web/WebSPA/Client/modules/catalog/catalog.component.ts:60]: As a JS newbie I would've thought this is redundant, so it looks like I have to know better 😅

File: src/Web/WebSPA/Client/modules/campaigns/campaigns.component.ts [hunk 2]
```diff
@@ -22,14 +22,14 @@ export class CampaignsComponent implements OnInit {
 
     ngOnInit() {
         if (this.configurationService.isReady) {
+            this.isCampaignDetailFunctionEnabled = this.configurationService.serverSettings.activateCampaignDetailFunction;
```
Comment (mvelosop) [inline src/Web/WebSPA/Client/modules/campaigns/campaigns.component.ts:25]: I think this module is related to the marketing functionality, so the whole thing should actually be removed

File: src/Web/WebSPA/appsettings.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿{
   "IdentityUrl": "http://localhost:5105",
-  "MarketingUrl": "http://localhost:5110",
+  "MarketingUrl": "http://localhost:5203",
```
Comment (mvelosop) [inline src/Web/WebSPA/appsettings.json:3]: Something more to remove.

**Review by mvelosop — CHANGES_REQUESTED** [review]
Thanks @cdemiguel, 

Just some minor comments, except for the need to remove Marketing (and Locations) features.

However it's just fine if you prefer to leave this PR for the version upgrade and then make another to remove Marketing and Locations

BTW, please sign the Contributor License Agreement above.

**Review by mvelosop — APPROVED** [review]

**Pull request merged**